### ISSUE TITLE
Horario de Entrega

### ISSUE THREAD

**Issue by romeromaxi** [MatiasMdelP/Al-Go-Oh#6]
Hola Tomás, como andas? 
El jueves nos encontramos para la entrega del tp a la hora del final?

Saludos

**Comment by tomasBustamante** [comment]
Sí, a partir de las 18:30 voy a estar. Si pueden un poco más tarde mejor porque creo que va a haber mucha gente.
